sympy/solvers/tests/test_solveset.py

```diff
@@ -709,21 +709,9 @@ def test_solveset_complex_tan():
 
 def test_solve_trig():
     from sympy.abc import n
-    assert solveset_real(sin(x), x) == \
```

Comment:
You can replace this (and others similarily) with `assert solveset(sin(x), x, S.Reals) == imageset(Lambda(n, pi*n), S.Integers)`